understanding/text-spacing.html

```diff
@@ -390,47 +313,41 @@ <h4>Languages and Scripts</h4>
                   			
                   <section id="results">
                      
-                     <h4>Results</h4>
+                     <h4>結果</h4>
                      
-                     <p id="langimpact">No adverse effects occurred. The following are the specific findings:</p>
+                     <p id="langimpact">悪影響はなかった。具体的な調査結果は次のとおりである:</p>
                      
                      <dl>
                         
-                        <dt>Character Spacing </dt>
+                        <dt>文字の間隔</dt>
                         
-                        <dd>Individual characters in words remained intact though they were spaced a bit further
-                           apart.
+                        <dd>単語の個々の文字は少し離れているが、完全なまま残った。
                         </dd>
                         
-                        <dt>Word Spacing </dt>
+                        <dt>単語の間隔</dt>
                         
-                        <dd>Words were spaced farther apart. In languages that do not have words (e.g. Japanese)
-                           applying word spacing had no effect. This is expected.
+                        <dd>単語はもっと離れていた。word spacing が適用された単語を含まない言語 (例 : 日本語) では、単語の間隔を適用しても効果はなかった。予想通りである。
                         </dd>
                         
-                        <dt>Line Height</dt>
+                        <dt>行の高さ</dt>
                         
-                        <dd>Changing line height did not separate diacritics from characters, nor did it adversely
-                           impact ascenders or descenders.
+                        <dd>行の高さを変更しても、発音区別符号と文字が区別されなくなったり、アセンダ及びディセンダに悪影響が及すこともなかった。
                         </dd>
                         
                      </dl>
                      
-                     <p>As previously discussed, the ability to read text with adjusted spacing is a user
-                        responsibility. This is true no matter the language.        
+                     <p>前述したように、間隔を調整してテキストを読む能力は利用者の責任である。これは言語に関係なく当てはまる。
                      </p>
                      
-                     <p>The SC's exception addresses cases where a text style property is not used in a language
-                        or script. In such cases, authors are only required  to ensure  relevant  properties
-                        do not break the layout.
+                     <p>達成基準の例外は、テキストのスタイルプロパティが言語又は表記体系で使用されていない場合に対処する。そのような場合、コンテンツ制作者は関連プロパティがレイアウトを壊さないことを保証することだけが求められる。
```

Comment:
「対処する」は変なのでは。ここの address は「扱う」くらいで良いのではないでしょうか。

```suggestion
                     <p>達成基準の例外は、テキストのスタイルプロパティが言語又は表記体系で使用されていない場合を扱う。そのような場合、コンテンツ制作者は関連プロパティがレイアウトを壊さないことを保証することだけが求められる。
```